openedx/core/djangoapps/content_libraries/api.py

```diff
@@ -452,7 +454,11 @@ def create_library(
         )
     except IntegrityError:
         raise LibraryAlreadyExists(slug)  # lint-amnesty, pylint: disable=raise-missing-from
-    CONTENT_LIBRARY_CREATED.send(sender=None, library_key=ref.library_key)
+    CONTENT_LIBRARY_CREATED.send_event(
+        content_library=ContentLibraryData(
+            library_key=ref.library_key
+        )
+    )
```

Comment:
@rpenido Put these `send_events` behind a temporary waffle flag and only trigger them if enabled.